examples/all_in_one/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,46 @@
+events {
+  worker_connections  1024;
+}
+
+http {
+    server_tokens off;
+    include       mime.types;
+    charset       utf-8;
+
+    log_format upstream_time_with_content_class_slo_info  '$remote_addr - $remote_user [$time_local] '
+                                                '"$request" $status $body_bytes_sent '
```

Comment:
I'd prefer to simplify the log format and drop the unused fields. Also instead of using `$request` it would be more readable to use `request-method="$request_method" request-uri="$request_uri" request_query="$args"`
and drop all the `uht`, `occ` etc fields. Lets try to keep it really simple as possible

Reply:
I've dropped the unused fields to simplify the log format. 

Regarding `$request` replacement, the proposed variables are not exact replacement, because nginx does not define any variable (afaik) which would capture used protocol as it is in `$request` (e.g. HTTP/1.1 in `GET /index.html?foo=bar HTTP/1.1`) .

Anyway, protocol version is not important at all for this example, so I've used the proposed variables to avoid any non-necessary  complexity of tailer RE.

https://github.com/seznam/slo-exporter/pull/4/commits/405c273a358be5e2da73147144ac90e713640fe3

Reply:
https://github.com/seznam/slo-exporter/pull/4#discussion_r458088070